### PR TITLE
feature: add heycar theme to storybook [HEYUI-2]

### PR DESCRIPTION
**Pull request by mwagdi**
### Description
- Removes `Docs` tab from `Guidelines` section
- Adds heycar font to storybook
- Replaces default storybook theme with heycar brand colors

### Ticket
[HEYUI-2](https://heycar.atlassian.net/browse/HEYUI-2)

### CODE REVIEW

File: .storybook/editor-theme.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+export const editorTheme = {
```
Comment (derek-heycar) [inline .storybook/editor-theme.js:1]: small correction.
we have a naming convention in the eslint.
here is the rule.
```
    'unicorn/filename-case': [
      'error',
      {
        cases: {
          camelCase: true,
          pascalCase: true,
        },
      },
    ],
 ```
it's means all javascript or TS files should be `camelCase ` or `pascalCase`
Maybe it's not working don't the `dot` folders.
It needs to check why.
If you have time are welcome.

Comment (derek-heycar) [inline .storybook/editor-theme.js:1]: small correction.
we have a naming convention in the eslint.
here is the rule.

```
    'unicorn/filename-case': [
      'error',
      {
        cases: {
          camelCase: true,
          pascalCase: true,
        },
      },
    ],
```

it's means all javascript or TS files should be  `camelCase`, `pascalCase `
Maybe it's not working don't the `dot` folders.
It needs to check why.
If you have time are welcome.

**Review by derek-heycar — CHANGES_REQUESTED** [review]

File: .storybook/editor-theme.js (hunk 1 above)
Comment (derek-heycar) [reply .storybook/editor-theme.js:1]: thanks ! good job. only one correction

**Review by derek-heycar — APPROVED** [review]

**Review by hey-igor — APPROVED** [review]
lgtm :)

**Pull request merged**